.github/workflows/build_flang.yml

```diff
@@ -104,8 +104,8 @@ jobs:
           CMAKE_OPTIONS="-DLLVM_TARGETS_TO_BUILD=${{ matrix.target }} \
             -DCMAKE_INSTALL_PREFIX=/usr/local \
             -DCMAKE_BUILD_TYPE=Release \
-            -DCMAKE_C_COMPILER=/usr/bin/${{ matrix.cc }} \
-            -DCMAKE_CXX_COMPILER=/usr/bin/${{ matrix.cpp }} \
+            -DCMAKE_C_COMPILER=/usr/local/bin/clang \
+            -DCMAKE_CXX_COMPILER=/usr/local/bin/clang++ \
```

Comment:
Does this mean that we will not be testing with gcc anymore?

Reply:
Yes and no.
Yes - we will not build `flang` with a native Linux compiler (gcc-10, clang-9, 10 or 11)
No - we are still using different compilers to build the llvm/clang in `classic-flang-llvm-project`. We download it as an artifact. So effectively - we still need one build per compiler in the `flang` project.

Reply:
So, are these following steps required?

```
          sudo apt install gcc-10 g++-10
          sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-9 10
          sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 20
          sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9 10
          sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-10 20
 ```

Reply:
Good point, @kiranchandramohan , I updated the PR removing these steps (and also the clang installation)
@pawosm-arm I used the opportunity to also use an env variable to store `/usr/local` prefix and not copy it all over the script. Thanks for the hint!